ensime_shared/ensime.py

```diff
@@ -130,13 +130,14 @@ def disable_plugin(self):
             If used from a secondary thread, this may need to use threadsafe
             Vim calls where available -- see :meth:`Editor.raw_message`.
         """
-        for path in self.runtime_paths:
+        for path in self.runtime_paths():
             self._vim.command('set runtimepath-={}'.format(path))
 
-    @property
+    # Tried making this a @property, with and without memoization, and it made
+    # plugin initialization misbehave in Neovim (only). WTF.
     def runtime_paths(self):  # TODO: memoize
         """All the runtime paths of ensime-vim plugin files."""
-        runtimepath = self._vim.eval('&runtimepath')
+        runtimepath = self._vim.options['runtimepath']
```

Comment:
> Do you prefer to merge it and fix the build after?

I guess it depends on how long it takes to get it sorted out. I'm continuing to work on top of this locally so it doesn't bother me to wait a bit, but this also passes tests for me locally and seems to be working for the two of us, some I'm fairly confident about it.

Here's [a sneak preview][1] of the rest on the way, has a few holes and things to finish but it basically works already!

[1]: https://github.com/ensime/ensime-vim/compare/master...ches:remove-server-lifecycle-from-client